Cargo.toml

```diff
@@ -12,3 +12,5 @@ license = "MIT"
 clap = { version = "4.3.12", features = ["derive", "cargo", "env"] }
 xdg = "2.5.0"
 bitcode = "0.6.3"
+log = "0.4.27"
+env_logger = "0.11.8"
```

Comment:
This is for adding some debug statements in log entries to make it easier to debug issues. I can remove if this is unwanted, but it really helps to understand what is happening when you call `RUST_LOG=trace comma`